assets/stylesheets/common/discourse-reactions.scss

```diff
@@ -340,3 +325,8 @@ nav.post-controls .actions button {
     height: 15px;
   }
 }
+
+nav.post-controls .actions .discourse-reactions-actions button:focus {
```

Comment:
do we need "button" ?

Reply:
I think so, as `discourse-reactions-actions` is div in which button's focus causes the issue

Reply:
my point is we don't want to target html elements directly, it adds specificity

Reply:
I recommend you read this article: https://css-tricks.com/specifics-on-css-specificity/

Reply:
Thanks for the article 😃 added the change in the latest commit